### PR TITLE
add a note of familiar-with for browsing contexts

### PR DESCRIPTION
**Pull request by siusin**
To address issue #951 .

### CODE REVIEW

File: sections/browsers.include [hunk 1]
```diff
@@ -282,7 +282,7 @@
 
 <h4 id="browsing-context-security">Security</h4>
 
-  A <a>browsing context</a> <var>A</var> is <dfn>familiar
+  A <a>browsing context</a> <var>A</var> is <dfn lt="familiar with|familiar">familiar
```
Comment (chaals) [inline sections/browsers.include]: If you make sure the whole `dfn` element is on one line you should not need to specify "familiar with" in the lt attribute, no?

**Review by chaals — CHANGES_REQUESTED** [review]
other than the line break, LGTM

File: sections/browsers.include (hunk 1 above)
Comment (siusin) [reply sections/browsers.include]: the term is also used as "familiar" in some paragraphs, so I use  "lt='...'" for the text needs to be in a conjugated form... any suggestion?

Comment (siusin) [reply sections/browsers.include]: @chaals The links of "familiar with" wont work once we remove "familiar with" from "lt", so I keep that and remove the line break in the latest commit.

**Review by chaals — APPROVED** [review]

**Pull request merged**